Makefile

```diff
@@ -24,7 +24,7 @@ RUN_FLAGS=-ti
 BUILD_IN_CONTAINER=true
 GO ?= env GO15VENDOREXPERIMENT=1 go
 GO_BUILD_INSTALL_DEPS=-i
-GO_BUILD_TAGS=-tags netgo
+GO_BUILD_TAGS=-tags 'netgo unsafe'
```

Comment:
I am not too happy about providing the unsafe tag globally, but I don't know how to provide tags for the library in isolation. For instance, compiling it separately before scope, results in the library being built again without `-tags unsafe`.